my-site.rb

```diff
@@ -0,0 +1,49 @@
+require "sinatra"
+require "./lib/scrabble.rb"
+
+class MySite < Sinatra::Base
+
+  get "/" do
+    @title = "Homepage"
+    erb :index
+  end
+
+  get "/score" do
+    @title = "Score One Word"
+    erb :score
+  end
+
+  post "/score" do
+    @title = "Word Score"
+    word_array = params[:word].split(" ")
+    word = word_array[0]
+    @valid_word = Scrabble::Scrabble.word?(word)
+    if @valid_word
+      bonus = params[:bonus] == "on"
+      @words = {}
+      @words[word] = {}
```

Comment:
You could probably DRY out this code where we setup the `@words` hash and add each word with its score and breakdown, by pulling lines 24-26 into their own method and calling that method here and on line 42.